packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -194,15 +236,7 @@ const baseConfig = (target) => {
 
                     mode === development && new webpack.NoEmitOnErrorsPlugin(),
 
-                    sdkReplacementPlugin(projectDir),
-
-                    pkg?.mobify?.extends && pkg?.mobify?.overridesDir
-                        ? caretOverrideReplacementPlugin(projectDir)
-                        : () => null,
-
-                    pkg?.mobify?.extends && pkg?.mobify?.overridesDir
-                        ? extendedTemplateReplacementPlugin(projectDir)
-                        : () => null,
```

Comment:
deleting the less performant (former) version of the plugin that used `webpack.NormalModuleReplacementPlugin` and had build times up to `2m 22s`